src/index.ts

```diff
@@ -111,14 +111,14 @@ const argv = yargs(process.argv.splice(2))
 	'si': {
 		alias: 's-index',
 		group: 'Static Options:',
-		desc: "Sends the specified directory index file. Use --no-si to disable directory indexing",
+		desc: 'Sends the specified directory index file. Use "--no-si" to disable directory indexing',
 		type: 'string',
 		default: 'index.html'
 	},
 	'sa': {
 		alias: 's-max-age',
 		group: 'Static Options:',
-		desc: "Set the max-age property of the Cache-Control header with a string in ms format",
+		desc: "Set the max-age property of the Cache-Control header with a string in milliseconds",
```

Comment:
This line suffered so much, that i already feel the guilt, but for one last time...
For what reason do you tell the user to write a "string in milliseconds" if it implies that your code accepts `--sa "some number"` format instead of just numbers? You have declared that this argument's type is `number`. So why?

Oh, and why do you keep writing that max-age is set in milliseconds if it's [impossible by definition](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control#response_directives)?

Reply:
Also, when describing something you refer to "it", so "set" becomes "sets".
```suggestion
		desc: 'Sets the max-age property of the Cache-Control header',
```